packages/esm-patient-orders-app/src/lab-results/lab-results.resource.ts

```diff
@@ -157,3 +158,69 @@ export async function updateOrderResult(
   }
   throw new Error('Failed to update order');
 }
+
+export function createObservationPayload(concept: LabOrderConcept, order: Order, values: any) {
```

Comment:
I'm always suspicious of explicitly using `any`, especially when the code here implies a rather specific shape, namely that `values` is that values is something like `Record<string, unknown>`, which would be much better to use.

We should likewise type the sub functions here as this helps ensure the correctness of the code.